src/client/components/layer-list/layer-list.vue

```diff
@@ -1,16 +1,20 @@
 <template>
-  <md-list>
+  <md-list ref="list" :style="{maxHeight: maxHeight + 'px'}">
     <li
-      v-for="layer in layers"
-      :key="layer.id"
+      v-for="(layer, index) in layers"
+      :key="layer.id + index"
```

Comment:
why is the `layer.id` not enough and is the index needed?

Reply:
Leftover from unfinished api and same id's. removed